src/Microsoft.AspNetCore.Razor.LanguageServer.Common/FilePathNormalizer.cs

```diff
@@ -3,10 +3,11 @@
 
 using System;
 using System.Net;
+using Microsoft.CodeAnalysis.Razor;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Common
 {
-    public sealed class FilePathNormalizer
+    public class FilePathNormalizer
     {
         public string Normalize(string filePath)
```

Comment:
Curious, can this method handle both absolute and relative paths?

Reply:
Yup